backend/db/V1_0__create_schema.sql

```diff
@@ -0,0 +1,22 @@
+CREATE SCHEMA IF NOT EXISTS brevity;
+
+CREATE TABLE IF NOT EXISTS brevity.user (
+    user_id INTEGER NOT NULL PRIMARY KEY,
+    openID_id INTEGER,
```

Comment:
how does this work? what do people use to log in/sign up?

Reply:
@howard021014  is working on the auth part and we can change this up. But we were planning to just use google auth or something to simplify logins so we don't have to do acc creation and stuff on our app.